<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="dbacea35-75a3-405b-8f69-74ebd5fe78a9" name="jfernandezt@JOSE-VAIO 2014-10-08 11:56:02" runUser="Jose-VAIO\jfernandezt" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="a02889c2-6b52-44bd-a167-3757354a0e9e">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="jfernandezt_JOSE-VAIO 2014-10-08 11_56_02">
      <DeploymentItem filename="C:\Users\jfernandezt\Desktop\APP TEMPORAL\APP_RANSA_NET_COLAS\PY_RANSA_NET_WCF\PY_RANSA_NET_EN\bin\Debug\PY_RANSA_NET_EN.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2014-10-08T11:56:02.0591044-05:00" queuing="2014-10-08T11:56:02.1001067-05:00" start="2014-10-08T11:56:02.1121074-05:00" finish="2014-10-08T11:56:02.5411319-05:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="ConsultaPedidoMovimiento" storage="c:\users\jfernandezt\desktop\app temporal\app_ransa_net_colas\py_ransa_net_wcf\py_ransa_net_ts\bin\debug\py_ransa_net_ts.dll" id="c18a5b4f-0de8-f588-f66f-1ed2055df807">
      <Execution id="22504296-0ce0-49c1-85c0-846e1c3ccf5a" />
      <TestMethod codeBase="C:/Users/jfernandezt/Desktop/APP TEMPORAL/APP_RANSA_NET_COLAS/PY_RANSA_NET_WCF/PY_RANSA_NET_TS/bin/Debug/PY_RANSA_NET_TS.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="PY_RANSA_NET_TS.Cls_PedidoTS, PY_RANSA_NET_TS, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="ConsultaPedidoMovimiento" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="c18a5b4f-0de8-f588-f66f-1ed2055df807" executionId="22504296-0ce0-49c1-85c0-846e1c3ccf5a" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="22504296-0ce0-49c1-85c0-846e1c3ccf5a" testId="c18a5b4f-0de8-f588-f66f-1ed2055df807" testName="ConsultaPedidoMovimiento" computerName="JOSE-VAIO" duration="00:00:00.0776770" startTime="2014-10-08T11:56:02.1801113-05:00" endTime="2014-10-08T11:56:02.5191307-05:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="22504296-0ce0-49c1-85c0-846e1c3ccf5a">
      <Output>
        <ErrorInfo>
          <Message>Test method PY_RANSA_NET_TS.Cls_PedidoTS.ConsultaPedidoMovimiento threw exception: 
System.Net.WebException: Error en el servidor remoto: (400) Solicitud incorrecta.</Message>
          <StackTrace>    en System.Net.HttpWebRequest.GetResponse()
   en PY_RANSA_NET_TS.Cls_PedidoTS.ConsultaPedidoMovimiento() en C:\Users\jfernandezt\Desktop\APP TEMPORAL\APP_RANSA_NET_COLAS\PY_RANSA_NET_WCF\PY_RANSA_NET_TS\Cls_PedidoTS.cs:línea 65
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>